client/src/shared/navigation/desktop.jsx

```diff
@@ -235,6 +237,7 @@ const DesktopNavBar = ({ user, isAuthenticated, isAdmin, logoutHandler }) => {
                     </Box>
                 </Row>
             </SC.StyledContainer>
+            {permitPath && <PermitsNav />}
```

Comment:
not the cleanest inclusion... how hard is it to match the current format? (I'm guessing the sub-nav makes it tricky)